ui/app/templates/components/mfa/mfa-form.hbs

```diff
@@ -50,7 +50,7 @@
               {{! template-lint-enable no-autofocus-attribute}}
             </div>
           {{else if (eq constraint.methods.length 1)}}
-            <p class="has-text-grey-400" data-test-mfa-push-instruction>
```

Comment:
I don't see this css helper anywhere else now that it's replaced with `-grey-light` - can we delete it?